test/run-tests.sh

```diff
@@ -11,5 +11,6 @@ then
   npm run test:unit || exit 1
   ./../scripts/uncovered-files
 else
-  ./parallel-similarity-test.sh
+  ./parallel-media-similarity-test.sh
+  ./parallel-text-similarity-test.sh
```

Comment:
Here it should be one or the other depending on whether we're running media similarity tests or text similarity tests.